add_matlab_bindings_at_startup.m

```diff
@@ -0,0 +1,3 @@
+startupFile = fopen(userpath+"/startup.m", 'a');
+fprintf(startupFile, "addpath "+ pwd + newline);
```

Comment:
Is it possible to exchange `pwd` with some command that determines the parent folder of `add_matlab_bindings_at_startup.m`? In python this is possible and it would allow us to execute this script from any working directory and lead to a more robust solution.